test/sanity_check.sh

```diff
@@ -6,10 +6,17 @@ set -e
 
 whitely echo Ping each host from the other
 
+function check_ping() {
```

Comment:
I don't think it is clear what this function is for, i.e. if someone reads this and thinks "why doesn't it just do the ping".  Ideally the purpose would be expressed in the name, but I can't think of a good one right now, so a comment is the next best thing.